.github/workflows/unit-test.yaml

```diff
@@ -0,0 +1,28 @@
+name: Unit Tests 
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    name: Run Unit Tests
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
Do we actually need Python 3.9 specifically? Or can we just rely on the system version?

Reply:
Correct, 3.9 is not necessary